src/renderer/services/dex.js

```diff
@@ -1465,6 +1468,251 @@ export default {
+
+            if (!res || !res.result || res.result.length < 120) {
+              resolve(commitState);
+              return;
+            }
+
+            commitState.quantityCommitted = u.fixed82num(res.result.substr(40, 16));
+            commitState.contributionHeight = Math.round(u.fixed82num(res.result.substr(56, 16)) * 100000000);
+            commitState.compoundHeight = Math.round(u.fixed82num(res.result.substr(72, 16)) * 100000000);
+            commitState.feesCollectedSnapshot = u.fixed82num(res.result.substr(88, 16));
+            commitState.feeUnitsSnapshot = u.fixed82num(res.result.substr(104, 16));
+            // console.log(`got commitState.feeUnitsSnapshot: ${commitState.feeUnitsSnapshot}`);
+
+            rpcClient.getBlock(commitState.contributionHeight)
+              .then((data) => {
+                commitState.contributionTimestamp = data.time;
+                resolve(commitState);
+              });
+          })
+          .catch((e) => {
+            reject(`Failed to fetch commit state. ${e.message}`);
+          });
+      } catch (e) {
+        reject(`Failed to fetch commit state. ${e.message}`);
+      }
+    });
+  },
+
+  fetchCommitDEXState() {
+    return new Promise((resolve, reject) => {
+      try {
+        const rpcClient = network.getRpcClient();
+        rpcClient.query({
+          method: 'getstorage',
+          params: [assets.DEX_SCRIPT_HASH, `${u.reverseHex(assets.APH)}fa`],
+        })
+          .then((res) => {
+            const dexState = {
+              totalUnitsContributed: 0,
+              lastAppliedFeeSnapshot: 0,
+              totalFeeUnits: 0,
+            };
+
+            if (res.result && res.result.length >= 48) {
+              dexState.totalUnitsContributed = u.fixed82num(res.result.substr(0, 16)) * 100000000;
+              dexState.lastAppliedFeeSnapshot = u.fixed82num(res.result.substr(16, 16));
+              dexState.totalFeeUnits = u.fixed82num(res.result.substr(32, 16));
+            }
+
+            rpcClient.query({
+              method: 'getstorage',
+              params: [assets.DEX_SCRIPT_HASH, `${u.reverseHex(assets.APH)}fc`],
+            })
+              .then((res) => {
+                if (!res || !res.result || res.result.length < 32) {
+                  dexState.totalFeesCollected = 0;
+                  resolve(dexState);
+                  return;
+                }
+
+                dexState.totalFeesCollected = u.fixed82num(res.result.substr(0, 16));
+                // console.log(`dexState.totalFeesCollected: ${dexState.totalFeesCollected}`);
+                resolve(dexState);
+              })
+              .catch((e) => {
+                reject(`Failed to fetch commit state. ${e.message}`);
+              });
+          })
+          .catch((e) => {
+            reject(`Failed to fetch commit state. ${e.message}`);
+          });
+      } catch (e) {
+        reject(`Failed to fetch commit state. ${e.message}`);
+      }
```

Comment:
this.$t() (three times above)

Reply:
I'm assuming these errors all get displayed in the UI by a toast or other handler. If they only show in the console, then no need to localize them.